src/RegService.jl

```diff
@@ -0,0 +1,41 @@
+module RegService
+
+using Sockets
+using Distributed
+using Pkg
+using Logging
+
+import ..RegEdit: register
+using ..Messaging
+
+const config = Dict{String,Any}()
+
+include("management.jl")
+
+function service(zsock::ReplySocket)
```

Comment:
Doc string here would be good

Reply:
Fixed in c2691afc8ec69b7ebc6e9cca5bdc4eaada988d50

Don't know why it doesn't show as outdated.